### PR TITLE
More rooms and orientations

### PR DESCRIPTION
**Pull request by ChielBruin**
Added more rooms to the level and implemented door and gate rotation.

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/model/level/Room.java [hunk 1]
```diff
@@ -59,6 +59,7 @@ public Room(String folder) {
 			RoomParser.importFile(folder, mazeTiles, entities, lights, 0, 0);	
 			setDoors();
 		} catch (IOException e) {
+			System.out.println(e.getMessage());
```
Comment (Taeir) [inline src/main/java/nl/tudelft/contextproject/model/level/Room.java]: Left over syso

**Comment by Taeir** [comment]
Looks good.

There is still one syso left. After that has been removed, this can be merged.

**Comment by ChielBruin** [comment]
@Taeir Fixed it!

**Comment by Geweldig** [comment]
Look good, merging.

**Pull request merged**